nixos/modules/services/mail/isync.nix

```diff
@@ -0,0 +1,72 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.isync;
+in {
+
+  options.services.isync = {
+    enable = mkEnableOption "Isync, a software to dispose your mailbox(es) as a local Maildir(s)";
+
+    install = mkOption {
```

Comment:
I'm a bit confused as to why there's both `enable` and `install` doing pretty much the same